### ISSUE TITLE
iOS: memories (videos, in this case) are marked as duplicates when they are not duplicates

### ISSUE THREAD

**Issue by sneak** [ente-io/ente#3490]
### Description

![image](https://github.com/user-attachments/assets/8c7c4adb-6a62-40af-bbd7-39f61b5aef6f)

see above.  two different videos, as can be seen in the thumbnail.

### Version

0.9.44

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - iOS

**Comment by ua741** [comment]
Can you long-press on the video and play both of them? What might have happened is that these videos were uploaded from different sources, and the logic for thumbnail generation might be different for each.

**Comment by sneak** [comment]
Yes, that was the issue.  Seems like the thumbnail generation needs to be unified.
